.travis.yml

```diff
@@ -26,11 +34,10 @@ after_success:
     # but we don't want travis to report a failure in the github UI just
     # because the coverage report failed to be published.
     - if [[ "$COVERAGE" == "true" ]]; then coveralls || echo "failed"; fi
-cache: apt
 notifications:
   webhooks:
     urls:
       - https://webhooks.gitter.im/e/4ffabb4df010b70cd624
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
-    on_start: false     # default: false
+    on_start: never     # options: [always|never|change] default: always
```

Comment:
It is to chose the notification that are sent to the webhook:
http://docs.travis-ci.com/user/notifications/#Webhook-notification